lib/rspec-console/exceptions.rb

```diff
@@ -0,0 +1,8 @@
+module RSpecConsole
+  class VersionError < StandardError
+    'Please use RSpec 2.9.10 or later'
```

Comment:
little confused, as any raise, imo, cant be described as "having no effect on the code". whether you prefer to retain it as expression or break it out into an object, however, has little effect on the code. but i like to break them out because i start to see common cases. ymmv.